pokemongo_bot/cell_workers/sleep_schedule.py

```diff
@@ -60,6 +64,22 @@ def _process_config(self):
         self.duration_random_offset = int(
             timedelta(
                 hours=duration_random_offset.hour, minutes=duration_random_offset.minute).total_seconds())
+        
+        self.wake_up_at_location = False
+        wake_up_at_location = self.config.get("wake_up_at_location", "")
```

Comment:
It is because in def work(), I had to check if wake_up_at_location is setup. I presumed it would not make a difference using a var or checking in config_file. Will fix.